src/config.ts

```diff
@@ -30,6 +35,18 @@ export const LOCALSTACK_ENDPOINT: string | undefined = loadOptionalEnvVar(
   "LOCALSTACK_ENDPOINT",
 );
 
+// Rate limiting
+
+export const lowRateLimiterConfiguration: TokenBucketConfiguration = {
+  capacity: 500,
```

Comment:
A maximum of 500 requests per minute is going to be what we ask our prototyping partners to follow for V1. When the Rate limiter is released we will actually be able to see whether it is a good starting point for our users.